### ISSUE TITLE
[Bug] Transparency Quality forced to Sharp

### ISSUE THREAD

**Issue by Kenny-Hui** [leezer3/OpenBVE#863]
### Description

Since the QuadTree PR is merged<?>, the transparency quality is now forced to "Sharp", despite the settings stating otherwise.
![image](https://user-images.githubusercontent.com/28094366/217477825-c1d00231-a70a-41da-915e-6e8e427d80a4.png)
This affects OpenBVE, ObjectViewer and RouteViewer, and causes some weirdness when using transparent texture.

### Reproduction

1. Launch OpenBVE/ObjectViewer/RouteViewer
2. View an object that utilize a transparent texture (Color key or png transparency)

## Route

Any.

## Train

Any.

### Related information 

* Operating system: Debian Testing
* Method of control: Keyboard

**Comment by leezer3** [comment]
I *hate* threading :(

TLDR:
The quad tree changes moved visibility updates to a separate thread.
However, OpenGL is fundamentally single threaded, and we can't upload textures to the graphics card from this background thread. (Well, you can play with hidden windows and stuff, but this doesn't always work well cross platform)
The visibility update though needs to have the texture bytes available in order to determine the transparency type.....
